tasks/3_is-boolean/is-boolean.js

```diff
@@ -0,0 +1,4 @@
+export function isBoolean(bool) {
+  if (bool === true || bool === false) return true;
+  else return false;
```

Comment:
Could this be simplified to:

`return typeof bool === 'boolean';` ?

Reply:
Changed.